README.md

```diff
@@ -2,347 +2,459 @@
-
-## Dev. tip
-When working on a feature in the Debezium based pipeline, you can replay old
-binlog updates by setting the system properties `database.offsetStorage` and
-`database.databaseHistory` to an old version of the history. In other words,
-you can start from history/offset pair A, add some events in OpenMRS (e.g.,
-add an Observation) to create some updates in the binlog. Now if you keep a
-copy of original version of A (before the changes) you can reuse it in future
-runs to replay the new events with no more interactions with OpenMRS.
+Once running you can access OpenMRS at <http://localhost:8099/openmrs/> using
+username "admin" and password "Admin123". The Docker image includes the required
+FHIR2 module and demo data.
 
-
-# Batch mode 
-The steps above for setting up a FHIR Store and a linked BigQuery dataset needs
-to be followed. Once it is done, and after `mvn install`, the pipeline can be
-run using a command like:
+If `docker-compose` fails, you may need to adjust file permissions.
 
 ```
-$ java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
-    org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:9018/openmrs \
-    --searchList=Patient,Encounter,Observation --batchSize=20 \
-   --targetParallelism=20 --fhirSinkPath=projects/PROJECT/locations/LOCATION/datasets/DATASET/ \
-   --sinkUserName=hapi --sinkPassword=hapi --fileParquetPath=tmp/TEST/ \
-   --openmrsUserName=admin --openmrsPassword=Admin123 `
-```
-The `searchList` argument accepts a comma separated list of FHIR search URLs.
-For example, one can use `Patient?given=Susan` to extract only Patient resources
-that meet the `given=Susan` criteria. You can also dump the output into Parquet
-files using the `--outputParquetBase` flag.
-
-If you prefer not to use a bundled jar (e.g., during development in an IDE) you
-can use the Maven exec plugin:
-```
-$ mvn exec:java -pl batch \
-    "-Dexec.args= --openmrsServerUrl=http://localhost:9020/openmrs  --searchList=Observation \
-    --batchSize=20 --targetParallelism=20 --fileParquetPath=tmp/TEST/ \
-    --fhirSinkPath=projects/PROJECT/locations/LOCATION/datasets/DATASET/ \
-    --sinkUserName=hapi --sinkPassword=hapi \
-     --openmrsUserName=admin --openmrsPassword=Admin123 "
+$ docker-compose -f openmrs-compose.yaml down -v
+$ chmod -R 755 ./utils
+$ docker-compose -f openmrs-compose.yaml up
 ```
-NOTE : In order to export data to a fhir sink , pass the '--fhirSinkPath' , 
-In order to  generate Parquet files, pass the '--fileParquetPath' argument 
-# Using Docker compose
-Alternatively you can spin up the entire pipeline using docker containers by running
 
-#### 1. Fire up OpenMRS (containing FHIR2 module)
+In order to see the demo data in OpenMRS you must rebuild the search index. In
+OpenMRS go to **Home > System Administration > Advanced Administration**. Under
+**Maintenance** go to **Search Index** then **Rebuild Search Index**.
+
+### Run HAPI FHIR server using Docker
```

Comment:
Please add a note that this is only needed if someone needs to test the "send to FHIR server" scenario, i.e., just to generate the data warehouse (i.e., the Parquet files) this is not needed.